packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -72,10 +73,11 @@ const validProjectName = (s) => {
     return regex.test(s) || 'Value can only contain letters, numbers, space and hyphens.'
 }
 
-const validAppExtensionNameRegex = /^(@[a-zA-Z0-9-_]+\/)?extension-[a-zA-Z0-9-_]+$/
 const validProjectAppExtensionName = (input) => {
-    if (!validAppExtensionNameRegex.test(input)) {
-        return 'The Application Extension name must follow the format @{namespace}/extension-{package-name} (namespace is optional).'
+    const result = validatePackageName(input)
+    if (!result.validForNewPackages) {
+        const errors = result.errors || []
+        return `Invalid package name: ${errors.join(', ')}`
```

Comment:
nit I feel it would be more clear to include "npm", since we are validating against npm guidelines/avaliability.

```suggestion
        return `Invalid npm package name: ${errors.join(', ')}`
```

Reply:
never mind, I see you have already mentioned this in the later change.